stdlib/messaging/activemq-artemis/src/main/ballerina/artemis/message.bal

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+# Represents ActiveMQ Artemis Message
+public type Message client object {
+
+    private MessageType messageType = TEXT;
+    private MessageConfiguration configuration;
+
+    public function __init(Session session, io:ReadableByteChannel | int | float | byte | boolean | string |
```

Comment:
I believe the format we've followed is to not have spaces around `|` with union types.

Reply:
It seems to be the formatting style in VSCode. Refer mail: https://groups.google.com/d/msg/ballerina-dev/Ayv1J9QdI0s/Bxm1PyB7GQAJ
I am using VSCode for formatting